go/http/api.go

```diff
@@ -147,11 +150,34 @@ func (api *APIImpl) AggregatedMetrics(w http.ResponseWriter, r *http.Request, ps
 	json.NewEncoder(w).Encode(responseMap)
 }
 
-// ThrottleApp forcibly marks given app as throttled. Future requests by this app will be denied.
+// ThrottleApp forcibly marks given app as throttled. Future requests by this app may be denied.
 func (api *APIImpl) ThrottleApp(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	appName := ps.ByName("app")
-	err := api.consensusService.ThrottleApp(appName)
+	var expireAt time.Time // default zero
+	var ttlMinutes int64
+	var ratio float64
+	var err error
+	if ps.ByName("ttlMinutes") == "" {
```

Comment:
Is this consistant with the API docs? 

```
- `/throttle-app/archive/0/0.3`: if already throttled, maintain same TTL and change ratio to `0.3` (`30%` refused). If not already throttled, TTL is one hour
```

Should we have a more explicit way to keep TTL while changing the ratio? Either different endpoints or a more explicit path argument?